web/static/scss/style.scss

```diff
@@ -0,0 +1,288 @@
+@font-face {
+	font-family: 'emulogic';
```

Comment:
I would prefer we do not host 4 different font files. Why do we need all of these? If you really need all of them, can we find a hosted solution?